src/config.cc

```diff
@@ -318,6 +324,67 @@ void Config::initFieldCallback() {
         return srv->storage_->SetDBOption(trimRocksDBPrefix(k),
                                           std::to_string(RocksDB.max_total_wal_size * MiB));
       }},
+      {"rocksdb.enable_blob_files", [](Server* srv, const std::string &k, const std::string& v)->Status {
+        if (!srv) return Status::OK();
+        std::string enable_blob_files = v == "yes" ? "true" : "false";
```

Comment:
@git-hulk currently, config parser could ignore case, i.e. YES will be changed into yes?

Reply:
Yes, for yes|no would ignore case.

Reply:
@caipengbo Need lowercase the value or use 

std::string opt_value = RocksDB.enable_blob_files ? "true":"false";